src/core/jwt/jwtTokenHandler.js

```diff
@@ -103,7 +103,13 @@ const jwtTokenHandlerFactory = function jwtTokenHandlerFactory({
                 if (response.status === 200) {
                     return response.json();
                 }
-                const error = new TokenError('Refresh-token expired', response);
+                if(response.status === 401){
+                    const error = new TokenError('Refresh-token expired', response);
+                    return Promise.reject(error);
+                }
+
+                let error = new Error('Unsuccessful token refresh');
```

Comment:
This is because it will be handled differently in test runner. TokenError has separate message, 401 code and not recoverable. Here we should show message for 500 error, 500 code and it is recoverable as before.